testplan/common/config/__init__.py

```diff
@@ -1,3 +1,19 @@
 """TODO."""
```

Comment:
Can we delete this on top of the change please? We consistently avoid __init__.py docstrings, lets remove a TODO (some IDEs pick it up).

Reply:
I'll just remove those lines for files touched in this PR, we can have another PR for overall fix.